Formula/redis@2.6.rb

```diff
@@ -0,0 +1,73 @@
+class Redis26 < Formula
+  desc "Persistent key-value database, with built-in net interface"
+  homepage "http://redis.io/"
+  url "http://download.redis.io/releases/redis-2.6.17.tar.gz"
+  sha256 "5a65b54bb6deef2a8a4fabd7bd6962654a5d35787e68f732f471bbf117f4768e"
+  head "https://github.com/antirez/redis.git", :branch => "2.6"
+
+  bottle do
+    cellar :any
+    sha256 "1c0c9565f242a1e34d171950a341e2f38372e49cc5420162c73cf2c39ef2c682" => :yosemite
+    sha256 "f4abb52ec0ef232b85aae9c0e3bbc6cfdfca3e43daeef38e46f3eaeb35644b45" => :mavericks
+    sha256 "4d936a5010e9b50f1c9c347192f11e0b8e365a1948455f6b27415d1a28e35b50" => :mountain_lion
+  end
+
+  keg_only :versioned_formula
+
+  conflicts_with "redis",
+                 :because => "Differing version of Homebrew/homebrew Redis"
+
+  def install
+    # Architecture isn't detected correctly on 32bit Snow Leopard without help
+    ENV["OBJARCH"] = MacOS.prefer_64_bit? ? "-arch x86_64" : "-arch i386"
+
+    # Head and stable have different code layouts
+    src = (buildpath/"src/Makefile").exist? ? buildpath/"src" : buildpath
+    system "make", "-C", src, "CC=#{ENV.cc}"
+
+    %w[benchmark cli server check-dump check-aof].each { |p| bin.install src/"redis-#{p}" }
```

Comment:
This is fine in it's current form. It would not affect existing installations.

`%w[benchmark cli server check-dump check-aof].each { |p| bin.install src/"redis-#{p}" }` would create executables `redis-benchmark redis-cli redis-server redis-check-dump redis-check-aof` in `/usr/local/Cellar/redis@2.6/2.6.17/bin` and then symlink them to `/usr/local/opt/redis@2.6/2.6.17/bin`.